cg/meta/transfer/external_data.py

```diff
@@ -5,210 +5,204 @@
-        return Path(self.destination_path % customer, lims_sample_id)
+        return Path(self.destination_path % self.customer_id, lims_sample_id)
 
-    def transfer_sample_files_from_source(self, dry_run: bool, ticket: str) -> None:
+    def transfer_sample_files_from_source(self, ticket: str, dry_run: bool = False) -> None:
         """Transfers all sample files, related to given ticket, from source to destination"""
-        cust: str = self.status_db.get_customer_id_from_ticket(ticket=ticket)
-        log_dir: Path = self.create_log_dir(ticket=ticket, dry_run=dry_run)
-        error_function: str = ERROR_RSYNC_FUNCTION.format()
-        Path(self.destination_path % cust).mkdir(exist_ok=True)
+        self._set_parameters(ticket=ticket, dry_run=dry_run)
+        LOG.debug(f"Transferring files for ticket {self.ticket} of customer {self.customer_id}")
+        log_dir: Path = self._create_log_dir()
+        self._get_destination_path().mkdir(exist_ok=True)
 
         command: str = RSYNC_CONTENTS_COMMAND.format(
-            source_path=self.get_source_path(customer=cust, ticket=ticket),
-            destination_path=self.get_destination_path(customer=cust),
+            source_path=self._get_source_path(),
+            destination_path=self._get_destination_path(),
         )
-        sbatch_parameters: Sbatch = Sbatch(
-            job_name=ticket + self.RSYNC_FILE_POSTFIX,
+        sbatch_parameters = Sbatch(
+            job_name=self.ticket + self.RSYNC_FILE_POSTFIX,
             account=self.account,
             number_tasks=1,
             memory=1,
             log_dir=str(log_dir),
             email=self.mail_user,
             hours=24,
             commands=command,
-            error=error_function,
+            error=ERROR_RSYNC_FUNCTION.format(),
         )
         self.slurm_api.set_dry_run(dry_run=dry_run)
-        sbatch_content: str = self.slurm_api.generate_sbatch_content(
-            sbatch_parameters=sbatch_parameters
-        )
-        sbatch_path: Path = Path(log_dir, ticket + self.RSYNC_FILE_POSTFIX + ".sh")
+        sbatch_content: str = self.slurm_api.generate_sbatch_content(sbatch_parameters)
+        sbatch_path = Path(log_dir, self.ticket + self.RSYNC_FILE_POSTFIX + ".sh")
         self.slurm_api.submit_sbatch(sbatch_content=sbatch_content, sbatch_path=sbatch_path)
-        LOG.info(
-            "The folder {src_path} is now being rsynced to hasta".format(
-                src_path=self.get_source_path(customer=cust, ticket=ticket)
-            )
+        LOG.info(f"The folder {self._get_source_path().as_posix()} is now being rsynced to hasta")
+
+    def _curate_sample_folder(self, sample_folder: Path) -> None:
+        """
+        Changes the name of the folder to the sample internal_id. If force is set to True,
+        replaces any previous folder.
+        Raises:
+            Exception if the sample from the folder is not present in statusdb.
+        """
+        # TODO: Implement dry-run
```

Comment:
```suggestion
```
Or do it in this PR or open issue.

Reply:
I will do it in this PR after I solve some issues